app/src/main/java/com/keylesspalace/tusky/db/AccountEntity.kt

```diff
@@ -69,6 +69,7 @@ data class AccountEntity(
     var activeNotifications: String = "[]",
     var emojis: List<Emoji> = emptyList(),
     var tabPreferences: List<TabData> = defaultTabs(),
+    var drawerPreferences: List<TabData> = defaultTabs(),
```

Comment:
I'm not sure what good defaults could be, but I think we should be able to set different ones for drawer and tabs,

Reply:
Fixed